version: "3.5"

services:
  plex:
    container_name: plex
    image: plexinc/pms-docker:1.41.3.9314-a0bfb8370
    hostname: ${APP_DOMAIN}
    restart: on-failure:5
    environment:
      # General
      - TZ=Europe/Paris
      # NGINX + LETSENCRYPT
      - VIRTUAL_PORT=32400
      - VIRTUAL_HOST=${APP_DOMAIN}
      - LETSENCRYPT_HOST=${APP_DOMAIN}
      # APP
      - PLEX_CLAIM=${PLEX_CLAIM}
    ports:
      - "32400:32400/tcp"
    networks:
      - proxy
    devices:
      - "/dev/dri:/dev/dri"
    volumes:
      - ./volumes/config:/config # plex database
      - ~/files:/data # library

networks:
  proxy:
    external: true
